tests/unit/models/gpflow/test_models.py

```diff
@@ -359,6 +360,54 @@ def test_find_best_model_initialization_improves_likelihood(
     npt.assert_array_less(post_init_loss, pre_init_loss)
 
 
+@random_seed
+def test_gaussian_process_regression_trajectory_sampler_returns_a_trajectory_sampler(
+    gpflow_interface_factory: ModelFactoryType, dim: int
```

Comment:
since this is for GPR only lets not use the factory at all, below use the GPR wrapper directly